scripts/setup.py

```diff
@@ -0,0 +1,27 @@
+import setuptools
+
+setuptools.setup(
+    name = "imgtool",
+    version = "0.1.1",
+    author = "The MCUboot commiters",
+    #author_email = "dev-mcuboot@lists.runtime.co",
+    author_email = "utzig@apache.org", #FIXME
```

Comment:
Why not remove the author email altogether and let people contact us via the GitHub issues?

Reply:
Removed, also updated version to `1.2.0` and fixed PEP8 issues.